zerver/lib/bugdown/__init__.py

```diff
@@ -693,7 +693,12 @@ def make_emoji(emoji_name, src, display_string):
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
-    elt.set("title", display_string)
+    if display_string[0] == ':':
+        # removes the colons at the ends and replaces space with underscore
+        elt.set("title", display_string[1:-1].replace("_", " "))
+    else:
+        # for unicode emojis
+        elt.set("title", display_string)
```

Comment:
I think that atleast an `if` statement is necessary in this function. If we pass the `display_string` without the `:`, the **alt** text doesn't turn out right. I tested calling it in different ways. But it doesn't seem to workout. So having an `if` is the best option.